src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -332,6 +332,87 @@ vs. ``it('image edit test' () => ...)``. As shown, the first example makes gramm
 prefix whereas the second message does not.``it()`` should describe what the individual test is doing and
 what it expects to happen.
 
+Differences between Frontend Unit Tests and End-to-End (E2E) Tests / FAQ
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+General introduction about testing and E2E/unit tests 
+
+
+What are E2E/unit tests designed for?
+.....................................
+
+E2E test:
+
+"Protractor is an end-to-end test framework for Angular and AngularJS applications.
+Protractor runs tests against your application running in a real browser,
+interacting with it as a user would." `(src) <http://www.protractortest.org/#/>`__
+
+It requires a fully functional system and tests the interation of all components
+of the application (Ceph, back-end, front-end).
```

Comment:
We wanted to document the status quo here, so the configuration of Grafana, Prometheus and the Alertmanager is currently not required to run the tests - so my suggestion would be to add it as soon as we have tests which require it.
I'm also not sure if it's necessary to explicitly say that the dashboard is activated (as I would think that this is clear to everyone ?).

"Each test will be run in a browser to simulate a user that uses the dashboard. This will automatically test the whole infrastructure that is required to run the tests." - This is already covered in line 347-348.